packages/augur-ui/src/modules/positions/selectors/select-reporting-winnings-by-market.ts

```diff
@@ -131,17 +123,14 @@ function sumClaims(
       },
     ];
   }
-  console.log("adding value", formatAttoRep(addedValue).formatted);
   marketsCollection.unclaimedRep = marketsCollection.unclaimedRep.plus(
     addedValue
   );
-  console.log("summing unclaimedRep", formatAttoRep(marketsCollection.unclaimedRep).formatted);
   return marketsCollection;
 }
 
 function isClaimable(marketId: string) {
   const market = selectMarket(marketId);
-  if (!market) {console.log("market is null", marketId); return false;}
   return (
```

Comment:
This will break if market is null or undefined.

I guess you meant only to remove the `console.log()` but removed the whole `if` statement